DEV_GUIDE.md

```diff
@@ -292,7 +292,7 @@ Do so by following these instructions.
 Start the `podman` machine as normal, then:
 
 ```shell
-echo 'mkdir -p /etc/containers/containers.conf.d && printf "[engine]\nservice_timeout=91\n" > /etc/containers/containers.conf.d/service-timeout.conf' && systemctl restart podman.socket |  podman machine ssh --username root --
+echo 'mkdir -p /etc/containers/containers.conf.d && printf "[engine]\nservice_timeout=91\n" > /etc/containers/containers.conf.d/service-timeout.conf && systemctl restart podman.socket' |  podman machine ssh --username root --
```

Comment:
Looks right, lets see if I can get podman machine running on fedora.  `podman machine start` fails and looks like you have to separately install a `podman-machine` package that sets things up properly, pull in qemu etc.

Reply:
okay LGTM! after running it

```
podman machine ssh --username root -- cat /etc/containers/containers.conf.d/service-timeout.conf
[engine]
service_timeout=91
```